pkg/starkapi/assets_api.go

```diff
@@ -106,4 +106,25 @@ func (assetsApi *AssetsApi) DeleteTag(asset domain.Asset, tagName string) error
 	}
 
 	return nil
-}
\ No newline at end of file
+}
+
+func (assetsApi *AssetsApi) CreateAuditLog(asset domain.Asset, log domain.AuditLog) error {
+	if len(asset.Ref) < 1 {
+		logger.Error("invalid asset in assets/CreateAuditLog. Asset does not have a ref.")
+		return errors.New(invalidAssetErrorStr)
+	}
+
+	data, err := json.Marshal(log)
+	if err != nil {
+		logger.Errorf("failed to marshal log with error : [%s]", err)
+		return err
+	}
+
+	url := fmt.Sprintf("%s/%s", assetsApi.baseUrl(), asset.Ref)
```

Comment:
is this the full and correct url for creating a change? I believe the url should be `/core/assets/$ref/history`

Reply:
![image](https://github.com/Stark-Tech-Group/stg-sdk-golang/assets/109103698/62c55ebe-c181-45f3-846f-4d6d85cd181b)